core/src/main/java/io/fabric8/maven/core/util/KubernetesResourceUtil.java

```diff
@@ -593,7 +593,7 @@ public int compare(Pod p1, Pod p2) {
     public static Date getCreationTimestamp(HasMetadata hasMetadata) {
         ObjectMeta metadata = hasMetadata.getMetadata();
         if (metadata != null) {
-            return parseTimestamp(metadata.getCreationTimestamp());
+            return parseTimestamp(metadata.getCreationTimestamp().getTime());
```

Comment:
`CreationTimestamp` is null for client generated objects. This will throw a `NullPointerException`.